concepts/submission_date.md

```diff
@@ -0,0 +1,65 @@
+# Submission Date
+
+This document will help you understand `submission_date`, why are standardizing on it, and how it
+differs from `activity_date`.
+
+## Table of Contents
+
+<!-- toc -->
+
+# What is Submission Date
+
+`submission_date` is the server time at which a ping is received from the client. We use it to
+partition many of our data sets.
+
+# Submission Delay
```

Comment:
Make a note of what pings this is useful for. Are all custom pings using pingsender? Additionally, mobile products do not have pingsender, so they suffer delay, given in [0].

[0] https://sql.telemetry.mozilla.org/queries/49867#134105

Reply:
fixed